### PR TITLE
chore: update readme with deployment steps

### PR DESCRIPTION
**Pull request by cctdaniel**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,14 +13,14 @@ First, make sure you have the [solana tool suite](https://docs.solana.com/cli/in
 installed on your machine. (The build depends on some C makefiles that are in the tool suite.)
 Make sure you have installed the same solana version that is being used in [CI](.github/workflows/docker.yaml)
 
-Then, simply run `cargo build` to compile the oracle program as a native binary, or `cargo build-bpf` to build a BPF binary 
+Then, simply run `cargo build` to compile the oracle program as a native binary, or `cargo build-bpf` to build a BPF binary
```
Comment (cctdaniel) [inline README.md:16]: ran by pre-commit

File: README.md [hunk 2]
```diff
@@ -148,3 +150,17 @@ root@pyth-dev# usermod -u 1002 -g 1004 -s /bin/bash pyth
 ```
 
 Finally, in docker extension inside VS Code click right and choose "Attach VS Code". If you're using a remote host in VS Code make sure to let this connection be open.
+
+## Deployment
+
+Oracle program upgrades are managed by the Pythian Council multisig. The steps are:
```
Comment (guibescos) [inline README.md:156]: The steps to deploy a new version

File: README.md [hunk 3]
```diff
@@ -148,3 +150,17 @@ root@pyth-dev# usermod -u 1002 -g 1004 -s /bin/bash pyth
...
+
+1. Create a release branch from `main`.
+2. Include binaries for both the Solana mainnet and Pythnet oracle programs (`pyth_oracle_solana.so` and `pyth_oracle_pythnet.so`).
```
Comment (guibescos) [inline README.md:159]: This is not really 2 it's more like : 
`1. Create a release branch from `main`. The release will contain binaries for ...`

File: README.md [hunk 4]
```diff
@@ -148,3 +150,17 @@ root@pyth-dev# usermod -u 1002 -g 1004 -s /bin/bash pyth
...
+3. [Install Solana CLI](https://docs.solana.com/cli/install-solana-cli-tools#use-solanas-install-tool) if not already installed.
+4. Set Solana config for the target network, e.g., devnet: `solana config set --url https://api.devnet.solana.com`.
+5. Execute `solana program write-buffer pyth_oracle_solana.so` to obtain the buffer address.
+6. Use `solana program set-buffer-authority` to assign the upgrade authority to the buffer address.
```
Comment (guibescos) [inline README.md:163]: I think this should explain how to find the upgrade authority.

File: README.md [hunk 5]
```diff
@@ -148,3 +150,17 @@ root@pyth-dev# usermod -u 1002 -g 1004 -s /bin/bash pyth
...
+
+## Deployment
+
+Oracle program upgrades are managed by the Pythian Council multisig. The steps are:
+
+1. Create a release branch from `main`.
+2. Include binaries for both the Solana mainnet and Pythnet oracle programs (`pyth_oracle_solana.so` and `pyth_oracle_pythnet.so`).
+3. [Install Solana CLI](https://docs.solana.com/cli/install-solana-cli-tools#use-solanas-install-tool) if not already installed.
+4. Set Solana config for the target network, e.g., devnet: `solana config set --url https://api.devnet.solana.com`.
+5. Execute `solana program write-buffer pyth_oracle_solana.so` to obtain the buffer address.
```
Comment (guibescos) [inline README.md:162]: It can be `pyth_oracle_solana.so` or `pyth_oracle_pythnet.so`

**Review by guibescos — APPROVED** [review]

**Pull request merged**